File: Mozu.SiteBuilder.UX.Admin/Scripts/app/model/CustomerAccountNote.js
/**
 * @class Taco.model.CustomerAccountNote
 */
Ext.define('Taco.model.CustomerAccountNote', {
    extend: 'Taco.core.data.Model',

    fields: [{
        "name": "id",
        "type": "int"
    }, {
        "name": "content",
        "type": "string"
    }, {
        "name": "createdBy",
        "type": "string"
    }, {
        "name": "createdOn",
        "type": "date",
        "dateFormat": "c"
    }],
    idProperty: 'id',
    
    proxy: {
        type: 'ajaxproxy',
        api: {
            read: '/admin/app/customers/notes/list',
            create: '/admin/app/customers/notes/create'
        },
        reader: {
            type: 'json',
            root: 'items',
            successProperty: 'success',
            messageProperty: 'message'
        },
        writer: {
            allowSingle: true,
            type: 'json'
        }
    }
});

File: Mozu.SiteBuilder.UX.Models/Customers/CustomerAccountNote.cs
using System;
using System.Runtime.Serialization;

namespace Mozu.SiteBuilder.UX.Models.Customers
{
    [DataContract]
    public class CustomerAccountNote
    {
        [DataMember(Name = "id")]
        public int? Id { get; set; }

        [DataMember(Name = "content")]
        public string Content { get; set; }

        [DataMember(Name = "createdOn")]
        public DateTime CreatedOn { get; set; }

        [DataMember(Name = "createdBy")]
        public string CreatedBy { get; set; }
    }
}